core/math/transform_2d.cpp

```diff
@@ -72,7 +72,7 @@ real_t Transform2D::get_skew() const {
 	return Math::acos(elements[0].normalized().dot(SGN(det) * elements[1].normalized())) - Math_PI * 0.5;
 }
 
-void Transform2D::set_skew(float p_angle) {
+void Transform2D::set_skew(real_t p_angle) {
```

Comment:
While you're at it, these could be made `const`. :)

Reply:
@aaronfranke by curiosity, is there any reason to not replace all float by real_t in the whole project?

Reply:
@fabriceci We don't want to just do this for everything because there are many places where we do explicitly want 32-bit floats (some examples that come to mind are colors, audio, arrays, rendering, and more). Transform2D is a place where we want everything to be `real_t`.

Reply:
All real_t parameters should be const ?

Reply:
We use `const` everywhere that we don't need the parameter to change (because it's constant). So, as long as we don't do `p_angle = something;`, we can make the argument `const`. It can be `float` or `real_t` or `Vector2` or anything else.

If you are not sure, just add `const` (make sure to do so both in the header and in the cpp file), and if it fails to compile then you know you can't put `const` there.

Reply:
Should all parameters changed to const when possible now.